### PR TITLE
Add unit tests for Media Helpers and Notification Sender files

### PR DESCRIPTION
**Pull request by tsukimi2**
(1) Added unit tests for server/lib/media-helpers.js and server/lib/notification-sender.js,
(2) refactored async-await usage in functions searchUserAndSetNotification and
searchVolunteerAndSetNotification in notificaiton-sender.js to facilitate unit
testing, and
(3) added function getAppRoot in server/tests/helpers.js

### CODE REVIEW

File: server/lib/notification-sender.js [hunk 1]
```diff
@@ -16,16 +16,17 @@ async function setNotification(id, notification, model){
...
-    users.forEach(function(user) {
-      user.roles.map(role => {
-        if (role === roleU){
-          if(whoUpdated !== undefined && whoUpdated !== user._id) setNotification(user._id, notification, User)
-          else setNotification(user._id, notification, User)
-        }
-      })
+export async function searchUserAndSetNotification(roleU, notification) {
+  const users = await User.find({})
+    .catch(err => {
```
Comment (jspaine) [inline server/lib/notification-sender.js]: Can you remove this catch, throw? Also in the function below

File: server/lib/media-helpers.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    afterEach(function() {
+      media_helpers.__ResetDependency__('mediaRoot')
+    })
+
+    describe('getPath', function() {
+      const getPath = media_helpers.__GetDependency__('getPath')
+
+      it('should return pathname of production environment of type string given a valid filename', function() {
+        const filename = 'testfile'
+        const expected = getAppRoot(__dirname) + media_root + '/' + filename
```
Comment (jspaine) [inline server/lib/media-helpers.spec.js]: I'm not sure of the point of this `getAppRoot`, you can just 
```
expect(actual).to.contain(`${mediaRoot}/${filename}`)
```

File: server/lib/media-helpers.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { 
+  default as media_helpers
```
Comment (jspaine) [inline server/lib/media-helpers.spec.js]: Try to stick to camelCase

File: server/lib/media-helpers.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        const expected = getAppRoot(__dirname) + media_root + '/' + filename
+        const actual = getPath(filename)
+
+        expect(actual).to.be.a('string')
+        expect(actual).to.equal(expected)
+      })
+    })		
+  })
+
+  context('node dev = debug', function() {
```
Comment (jspaine) [inline server/lib/media-helpers.spec.js]: node env = dev?

**Review by jspaine — COMMENTED** [review]
Looks good, I added some minor suggestions.

**Comment by tsukimi2** [comment]
Committed the suggested changes.  However, I just did git commit and git push -u origin <my-branch-name> instead of
git commit -amend
git push --force

Is the amendment of the pull request still going to work?

**Comment by jspaine** [comment]
Thanks!

Yeah that's fine, I'll squash them into one commit and merge it.

**Pull request merged**